<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MemoryTask</class>
 <widget class="QWidget" name="MemoryTask">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QLabel" name="strategyLabel">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelRequestDescr">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Описание заявки</string>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="acceptRequest">
         <property name="text">
          <string>Подтвердить</string>
         </property>
         <property name="shortcut">
          <string>Alt+Return</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="resetState">
         <property name="text">
          <string>Сбросить</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+Z</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="1">
      <widget class="QLabel" name="labelFreeBlocks">
       <property name="text">
        <string>Список свободных блоков памяти</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="labelMemBlocks">
       <property name="text">
        <string>Список блоков памяти ОС</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listMemBlocks">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QListWidget" name="listFreeBlocks">
       <property name="font">
        <font>
         <family>Consolas</family>
        </font>
       </property>
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::InternalMove</enum>
       </property>
       <property name="defaultDropAction">
        <enum>Qt::MoveAction</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
